model-api/build.gradle.kts

```
@@ -30,6 +30,7 @@ kotlin {
         }
         val commonTest by getting {
             dependencies {
+                api(project(":model-datastructure"))
```

```suggestion
                implementation(project(":model-datastructure"))
```
`api` dependencies are used to declare that the dependency needs to be rexported.
We do not need this here as it is a test source set that is not used anywhere else.